Components/Alert/alert.html

```diff
@@ -85,13 +84,32 @@ <h3 class="drawer-header">Get Started</h3>
         </div>
         <hr class="hr-line drawer-line" />
         <div class="drawer-container component-drawer-container">
-          <p class="drawer-inner-header"><a class="drawer-links" href="../Alert/alert.html">Alert</a></p> 
-          <p class="drawer-inner-header"><a class="drawer-links" href="../Avatar/avatar.html">Avatar</a></p> 
-          <p class="drawer-inner-header"><a class="drawer-links" href="../Badge/badge.html">Badge</a></p>
-          <p class="drawer-inner-header"><a class="drawer-links" href="../Button/button.html">Button</a></p> 
-          <p class="drawer-inner-header"><a class="drawer-links" href="../Card/card.html">Card</a></p>
-          <p class="drawer-inner-header"><a class="drawer-links" href="../Image/image.html">Image</a></p> 
-          <p class="drawer-inner-header"><a class="drawer-links" href="../Input/input.html">Input</a></p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Alert/alert.html">Alert</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Avatar/avatar.html">Avatar</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Badge/badge.html">Badge</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Button/button.html">Button</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Card/card.html">Card</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Image/image.html">Image</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Input/input.html">Input</a>
+          </p>
+          <p class="drawer-inner-header">
+            <a class="drawer-links" href="../Topography/topography.html"
```

Comment:
Rename the component folder and file to `Typography` and `typography` here as well as at all other places in this PR.